### ISSUE TITLE
pytorch - autograd

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#889]
# Reference
- [Autograd：自动求导](https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html#sphx-glr-beginner-blitz-autograd-tutorial-py)
# Brief

```
x = torch.ones(2, 2, requires_grad=True)
y = x + 2
z = y * y * 3
out = z.mean()
out.backward()
x.grad
```

```
tensor([[4.5000, 4.5000],
        [4.5000, 4.5000]])
```
Inputs | Outputs | Gradients
-- | -- | --
![image](https://user-images.githubusercontent.com/2216970/101307335-2e61d700-3882-11eb-8cb0-5be50fd4dc53.png) | ![image](https://user-images.githubusercontent.com/2216970/101307477-87ca0600-3882-11eb-9c13-001faaaffe3c.png) | ![image](https://user-images.githubusercontent.com/2216970/101307368-3e79b680-3882-11eb-8d88-33579743af86.png)

Function | ![image](https://user-images.githubusercontent.com/2216970/101306849-ef7f5180-3880-11eb-9ec4-62102694b69a.png) 
-- | -- 
Xi - Gradients | ![image](https://user-images.githubusercontent.com/2216970/101306888-058d1200-3881-11eb-88ef-74ad45ec30b8.png) | ![image](https://user-images.githubusercontent.com/2216970/101306939-2f463900-3881-11eb-8bf9-bc30477c12d3.png)
All - Gradients | ![image](https://user-images.githubusercontent.com/2216970/101307030-76342e80-3881-11eb-85f8-78575165c68f.png)

**Comment by junxnone** [comment]
#311
